circuits/cpp/src/aztec3/constants.hpp

```diff
@@ -43,13 +43,18 @@ constexpr size_t PRIVATE_DATA_SUBTREE_INCLUSION_CHECK_DEPTH = PRIVATE_DATA_TREE_
 constexpr size_t NULLIFIER_SUBTREE_DEPTH = 3;
 constexpr size_t NULLIFIER_SUBTREE_INCLUSION_CHECK_DEPTH = NULLIFIER_TREE_HEIGHT - NULLIFIER_SUBTREE_DEPTH;
 
+constexpr size_t L1_TO_L2_MSG_SUBTREE_DEPTH = 4;
```

Comment:
Would be nice to have `NUMBER_OF_L1_L2_MESSAGES_PER_ROLLUP` located up heere as well, otherwise might easily be missed and one could update one but not the other, would be fine to add a note that they are related with `2**L1_TO_L2_MSG_SUBTREE_DEPTH = NUMBER_OF_L1_L2_MESSAGES_PER_ROLLUP`